packages/design-system/src/components/Table/TableRow.tsx

```diff
@@ -1,9 +1,23 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 
-export const TableRow = ({ children, _isTableHeadChild, ...tableRowProps }) => {
+export interface TableRowProps {
+  /**
+   * The table row contents, usually `TableCell`.
+   */
+  children?: React.ReactNode;
+  /**
+   * @hide-prop This gets set from the parent `TableHead`
+   */
+  _isTableHeadChild?: boolean;
```

Comment:
`_stackable?: boolean;` is in the definition file but not here. Not sure if it's needed.